Date: Sun, 2 Feb 2003 16:53:39 +0100
From: Guillaume Morin <gemorin@debian.org>;
To: Debian Bug Tracking System <submit@bugs.debian.org>
Subject: xsl/country/fr.xsl contains non UTF-8 characters
Message-ID: <20030202155339.GA1518@siri>

fr.xsl contains some ISO8859-1 characters, which make fr.xsl unusable.
They are inside a comment.  I've chosen not to replace them by their
UTF-8 counterparts to make the comment more readable.

--- xml-resume-library/xsl/country/fr.xsl~      2003-02-02 16:43:42.000000000 +0100
+++ xml-resume-library/xsl/country/fr.xsl	2003-02-02 16:46:21.000000000 +0100
@@ -37,7 +37,7 @@
 
 <!-- French language notes (correct us if we're wrong:
  * Not all the words are capitalized in titles, only the first one. 
- * It's Téléphone privé, not privé Téléphone
+ * It's T'el'ephone priv'e, not priv'e T'el'ephone
 -->
 
   <xsl:param name="resume.word">Curriculum Vitae</xsl:param>

